rust/src/write.rs

```diff
@@ -290,6 +298,30 @@ impl WriteOptions {
     pub fn create<W: Write + Seek>(self, w: W) -> McapResult<Writer<W>> {
         Writer::with_options(w, self)
     }
+
+    /// Specifies whether to calculate and write CRCs for chunk records. This is on by default.
+    pub fn calculate_chunk_crcs(mut self, calculate_chunk_crcs: bool) -> Self {
```

Comment:
wondering if there are any commonly-used ways of reducing boilerplate, like a macro to make a setter for the builder pattern?

Reply:
we could certainly add a macro - I can take a look at this in a later PR.